cmd/beacon-chain/jwt/jwt.go

```diff
@@ -0,0 +1,69 @@
+package jwt
+
+import (
+	"encoding/hex"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/prysmaticlabs/prysm/cmd"
+	"github.com/prysmaticlabs/prysm/runtime/tos"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+var log = logrus.WithField("prefix", "jwt")
+
+// Commands for interacting with jwt tokens that support the beacon node.
+var Commands = &cli.Command{
+	Name:     "jwt",
+	Category: "jwt",
+	Usage:    "defines commands for interacting with jwt tokens that support the beacon node",
+	Subcommands: []*cli.Command{
+		{
+			Name:        "generate-jwt-secret",
+			Description: `creates a random 32 byte hex string in a plaintext file within the root /prysm directory`,
+			Flags:       cmd.WrapFlags([]cli.Flag{}),
```

Comment:
It could be good to take in a single flag perhaps called --output-file with shorthand `-o`. By default, it will just output it in the current working directory

Reply:
I didn't see a way to specify shorthand within the flag object, or an example of specifying shorthand... should we add another command dedicated to the shorthand notation? An additional flag feels like it could be noisy (I tend to bias towards "verbose and explicit" over "short and clever")